frontend/components/KnownOverallProgress.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { PropTypes } from 'react';
+
+import styles from './KnownOverallProgress.styl';
+
+const LOADING_IMAGES = '...';
+const LOADING_KNOWN = '...';
+
+export default function KnownOverallProgress({
```

Comment:
Nit: Other components declare the component functions as
```
const KnownOverallProgress = (props) => ();
```
then export them at the bottom as 
```
export default KnownOverallProgress;
```
I don't have a strong preference for one style over the other but we should probably be consistent one way or the other